### PR TITLE
[DO NOT MERGE] Test OV build

### PR DESCRIPTION
**Pull request by artanokhov**

### CODE REVIEW

**Comment by artanokhov** [comment]
@conda-forge-admin, please rerender

**Comment by artanokhov** [comment]
@xhochy @conda-forge/help-c-cpp 
Hello, folks
Could you please help to investigate the reason of strange errors that we have now:

`
  ERROR (libopenvino-auto-batch-plugin,lib/openvino-2023.2.0/libopenvino_auto_batch_plugin.so): lib/libopenvino.so.2320 not found in any packages
  ERROR (libopenvino-auto-batch-plugin,lib/openvino-2023.2.0/libopenvino_auto_batch_plugin.so): lib/libstdc++.so.6 not found in any packages
  ERROR (libopenvino-auto-batch-plugin,lib/openvino-2023.2.0/libopenvino_auto_batch_plugin.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libm.so.6 not found in any CDT/compiler package, nor the whitelist?!
  ERROR (libopenvino-auto-batch-plugin,lib/openvino-2023.2.0/libopenvino_auto_batch_plugin.so): lib/libgcc_s.so.1 not found in any packages
  ERROR (libopenvino-auto-batch-plugin,lib/openvino-2023.2.0/libopenvino_auto_batch_plugin.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libpthread.so.0 not found in any CDT/compiler package, nor the whitelist?!
  ERROR (libopenvino-auto-batch-plugin,lib/openvino-2023.2.0/libopenvino_auto_batch_plugin.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 not found in any CDT/compiler package, nor the whitelist?!
  ERROR (libopenvino-auto-batch-plugin,lib/openvino-2023.2.0/libopenvino_auto_batch_plugin.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/ld-linux-x86-64.so.2 not found in any CDT/compiler package, nor the whitelist?!
WARNING (libopenvino-auto-batch-plugin): run-exports library package local/linux-64::libopenvino==2023.2.0=h59595ed_1 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (libopenvino-auto-batch-plugin): dso library package conda-forge/linux-64::libstdcxx-ng==13.2.0=h7e041cc_3 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
Traceback (most recent call last):
  File "/opt/conda/bin/conda-mambabuild", line 10, in <module>
    sys.exit(main())
  File "/opt/conda/lib/python3.10/site-packages/boa/cli/mambabuild.py", line 256, in main
    call_conda_build(action, config)
  File "/opt/conda/lib/python3.10/site-packages/boa/cli/mambabuild.py", line 228, in call_conda_build
    result = api.build(
  File "/opt/conda/lib/python3.10/site-packages/conda_build/api.py", line 254, in build
    return build_tree(
  File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 3789, in build_tree
    packages_from_this = build(
  File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2877, in build
    newly_built_packages = bundlers[pkg_type](output_d, m, env, stats)
  File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2004, in bundle_conda
    files = post_process_files(metadata, initial_files)
  File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 1815, in post_process_files
    post_build(m, new_files, build_python=python)
  File "/opt/conda/lib/python3.10/site-packages/conda_build/post.py", line 1819, in post_build
    check_overlinking(m, files, host_prefix)
  File "/opt/conda/lib/python3.10/site-packages/conda_build/post.py", line 1644, in check_overlinking
    return check_overlinking_impl(
  File "/opt/conda/lib/python3.10/site-packages/conda_build/post.py", line 1621, in check_overlinking_impl
    raise OverLinkingError(overlinking_errors)
conda_build.exceptions.OverLinkingError: overlinking check failed 
['  ERROR (libopenvino-auto-batch-plugin,lib/openvino-2023.2.0/libopenvino_auto_batch_plugin.so): lib/libopenvino.so.2320 not found in any packages', '  ERROR (libopenvino-auto-batch-plugin,lib/openvino-2023.2.0/libopenvino_auto_batch_plugin.so): lib/libstdc++.so.6 not found in any packages', '  ERROR (libopenvino-auto-batch-plugin,lib/openvino-2023.2.0/libopenvino_auto_batch_plugin.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libm.so.6 not found in any CDT/compiler package, nor the whitelist?!', '  ERROR (libopenvino-auto-batch-plugin,lib/openvino-2023.2.0/libopenvino_auto_batch_plugin.so): lib/libgcc_s.so.1 not found in any packages', '  ERROR (libopenvino-auto-batch-plugin,lib/openvino-2023.2.0/libopenvino_auto_batch_plugin.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libpthread.so.0 not found in any CDT/compiler package, nor the whitelist?!', '  ERROR (libopenvino-auto-batch-plugin,lib/openvino-2023.2.0/libopenvino_auto_batch_plugin.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 not found in any CDT/compiler package, nor the whitelist?!', '  ERROR (libopenvino-auto-batch-plugin,lib/openvino-2023.2.0/libopenvino_auto_batch_plugin.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/ld-linux-x86-64.so.2 not found in any CDT/compiler package, nor the whitelist?!']
##[error]Bash exited with code '1'.
`

We noticed the first such errors for conda-forge/openvino-feedstock/pull/60, when preparing fresh release.
I created this PR for previous OpenVINO release 2023.2.0 (which was already built successfully) by simply increasing the build number and re-rendering. So it looks like these errors are caused by the build environment and not by our changes.
Could you please take a look into this?

If needed I can submit an issue.

**Comment by artanokhov** [comment]
@conda-forge-admin, please ping conda-forge/help-c-cpp

File: .gitignore
Comment (hmaarrfk) [inline .gitignore:1]: Try to revert the changes to this file in particular.

**Comment by xhochy** [comment]
I think that this is a general conda-build bug. I'm seeing similar issues in Tensorflow and also came around this PR https://github.com/conda-forge/libsecp256k1-feedstock/pull/1 which fixes the issue by downgrading `conda-build`.

**Comment by hmaarrfk** [comment]
I think we have a way to add these kinds of constraints in `conda-forge.yml`

**Comment by ilya-lavrenov** [comment]
Should be fixed in https://github.com/conda-forge/openvino-feedstock/pull/61
